boards/riscv/rv32m1_vega/Kconfig.defconfig

```diff
@@ -91,6 +91,15 @@ if BT
 config BT_CTLR
 	default y
 
+#TODO: Resolve the complete non-BLE support for crypto CAU3 firmware/driver
+#config HAS_RV32M1_CAU3
+#	bool
+#	default y if BT_CTLR_CRYPTO && !BT_CTLR_LE_ENC_SUPPORT
+
+config HAS_RV32M1_CAU3_BLE
+	bool
+	default y if BT_CTLR_CRYPTO && BT_CTLR_LE_ENC_SUPPORT
```

Comment:
default y if BT_CTLR_CRYPTO **||** BT_CTLR_LE_ENC_SUPPORT

Reply:
Certainly not || between the two.
CAU3_BLE driver is only for the LE_ENC_SUPPORT which requires AES_CCM.
The scenario where BT_HOST_CRYPTO && BT_CTLR_LE_ENC_SUPPORT is generally not valid. That is why I wanted it excluded.